src/components/QuickGrid/DataSelectors.ts

```diff
@@ -8,34 +8,31 @@ const getGroupBy = (state: IQuickGridState, props: IQuickGridProps) => props.gro
 const getExpandedRows = (state: IQuickGridState, props: IQuickGridProps) => state.expandedRows;
 const getSortColumn = (state: IQuickGridState, props: IQuickGridProps) => state.sortColumn;
 const getSortDirection = (state: IQuickGridState, props: IQuickGridProps) => state.sortDirection;
+const getGroupBySortColumn = (state: IQuickGridState, props: IQuickGridProps) => state.groupBySortColumn;
+const getGroupBySortDirection = (state: IQuickGridState, props: IQuickGridProps) => state.groupBySortDirection;
 const getColumns = (state: IQuickGridState, props: IQuickGridProps) => props.columns;
 
-const sortRows = (rows: Array<any>, sortColumn: string, sortDirection: string, groupedColumn: Array<string>, columns: Array<GridColumn>) => {
-    let column = columns.filter(x => x.valueMember === sortColumn)[0];
-    let sortedChain = _(rows).chain();
-
-    if (sortDirection && sortColumn) {
-        if (column && column.sortByValueGetter) {
-            sortedChain = sortedChain.sortBy((row) => column.sortByValueGetter(row, sortDirection));
+const sortRows = (rows: Array<any>, sortColumn: string, sortDirection: string, groupedColumn: Array<string>, columns: Array<GridColumn>, groupBySortColumn: string, groupBySortDirection: string) => {
+    const columnSortDir: string = sortDirection === 'DESC' ? 'desc' : 'asc';
```

Comment:
I have added SordDirection enum, it's definitely better than using string definitions in all interfaces